global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.4.23260.5",
+    "version": "8.0.100-preview.5.23267.7",
```

Comment:
Is this .NET SDK version update still needed for this PR?

We typically update .NET SDK version in separate PRs and only to officially released versions.